specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/networkData.json

```diff
@@ -0,0 +1,361 @@
+              "enum": [
+                "Available",
+                "Not available"
+              ],
+              "x-ms-enum": {
+                "name": "trafficDataState",
+                "modelAsString": true,
+                "values": [
+                  {
+                    "value": "Available",
+                    "description": "The Azure monitor agent is installed on the vm"
+                  },
+                  {
+                    "value": "Not available",
+                    "description": "the Azure monitor agent is not installed on the vm"
+                  }
+                ]
+              }
+            },
+            "inboundConnectedResources": {
+              "$ref": "#/definitions/ActualTrafficConnectedResources"
+            },
+            "outboundConnectedResources": {
+              "$ref": "#/definitions/ActualTrafficConnectedResources"
+            }
+          }
+        },
+        "unusedPorts": {
+          "readOnly": true,
+          "type": "object",
+          "description": "The unused ports details",
+          "properties": {
+            "inboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            },
+            "outboundConnectedResources": {
+              "$ref": "#/definitions/ConnectedResources"
+            }
+          }
+        }
+      }
+    },
+    "ConnectedResources": {
+      "readOnly": true,
+      "type": "object",
+      "description": "The connections details",
+      "properties": {
+        "tcpPortsCount": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "The allowed connections tcp ports count"
+        },
+        "udpPortsCount": {
+          "readOnly": true,
+          "type": "integer",
+          "description": "The allowed connections udp ports count"
+        },
+        "tcpPorts": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The allowed connections tcp ports"
+        },
+        "udpPorts": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The allowed connections udp ports"
+        }
+      }
+    },
+    "ActualTrafficConnectedResources": {
+      "readOnly": true,
+      "type": "object",
+      "description": "The actual traffic details",
+      "properties": {
+        "trafficVolume": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The actual traffic volume between the actual traffic start time to the actual traffic end time"
+        },
+        "connectionsDetails": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConnectionDetails"
+          }
+        }
+      }
+    },
+    "ConnectionDetails": {
+      "type": "object",
+      "description": "Details of the actual traffic connections",
+      "properties": {
+        "portAndProtocol": {
```

Comment:
>portAndProtocol [](start = 9, length = 15)

is there a benefit to combining these into one property instead of having a "port" and "protocol" property?  This requires the user to parse the value

Reply:
This is a standard way to describe this tuple in the networking world (just like you say "10.2.3.0/16" to describe a subnet)